pkg/cloudprovider/providers/gce/gce_instances.go

```diff
@@ -35,13 +35,46 @@ import (
 	"k8s.io/kubernetes/pkg/cloudprovider"
 )
 
+const (
+	zoneKey     = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
Is it possible to import this label from the original place? Like https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go#L21.